Python 3.6.3 (default, Oct  3 2017, 21:45:48) 
[GCC 7.2.0] on linux
Type "copyright", "credits" or "license()" for more information.
>>> 
==================== RESTART: /home/cs2017a111/ls10tw.py ====================
>>> 
==================== RESTART: /home/cs2017a111/ls10tw.py ====================
Traceback (most recent call last):
  File "/home/cs2017a111/ls10tw.py", line 18, in <module>
    api.update_status(status ="Hello Everyone !")
  File "/usr/local/lib/python3.6/dist-packages/tweepy/api.py", line 194, in update_status
    )(post_data=post_data, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/tweepy/binder.py", line 245, in _call
    return method.execute()
  File "/usr/local/lib/python3.6/dist-packages/tweepy/binder.py", line 229, in execute
    raise TweepError(error_msg, resp, api_code=api_error_code)
tweepy.error.TweepError: [{'code': 187, 'message': 'Status is a duplicate.'}]
>>> 
==================== RESTART: /home/cs2017a111/ls10tw.py ====================
Traceback (most recent call last):
  File "/home/cs2017a111/ls10tw.py", line 18, in <module>
    api.update_status(status ="Hello Everyone !")
  File "/usr/local/lib/python3.6/dist-packages/tweepy/api.py", line 194, in update_status
    )(post_data=post_data, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/tweepy/binder.py", line 245, in _call
    return method.execute()
  File "/usr/local/lib/python3.6/dist-packages/tweepy/binder.py", line 229, in execute
    raise TweepError(error_msg, resp, api_code=api_error_code)
tweepy.error.TweepError: [{'code': 187, 'message': 'Status is a duplicate.'}]
>>> 
==================== RESTART: /home/cs2017a111/ls10tw.py ====================
>>> 
==================== RESTART: /home/cs2017a111/ls10tw.py ====================
Traceback (most recent call last):
  File "/home/cs2017a111/ls10tw.py", line 18, in <module>
    api.update_status(status ="Hello budies!")
  File "/usr/local/lib/python3.6/dist-packages/tweepy/api.py", line 194, in update_status
    )(post_data=post_data, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/tweepy/binder.py", line 245, in _call
    return method.execute()
  File "/usr/local/lib/python3.6/dist-packages/tweepy/binder.py", line 229, in execute
    raise TweepError(error_msg, resp, api_code=api_error_code)
tweepy.error.TweepError: [{'code': 187, 'message': 'Status is a duplicate.'}]
>>> 
